python_modules/dagster/dagster/_core/execution/plan/compute_generator.py

```diff
@@ -257,7 +259,7 @@ def validate_and_coerce_op_result_to_iterator(
             "value. Check out the docs on logging events here: "
             "https://docs.dagster.io/concepts/ops-jobs-graphs/op-events#op-events-and-exceptions"
         )
-    elif isinstance(result, AssetCheckResult):
+    elif isinstance(result, (AssetCheckResult, ObserveResult)):
```

Comment:
is this correct? comment-worthy if it is

Reply:
yes, I added a comment